src/controllers/file.ts

```diff
@@ -30,6 +32,7 @@ export default class UploadController {
     // Add the file to IPFS
     const { cid } = await ipfs.add(fileContent)
     await ipfs.pin.add(cid)
+    await backupCID(cid.toString())
```

Comment:
Should we `await` on it? Maybe let's `void` or `timeout` so we don't delay file return?

Reply:
Sure, resolved in [d7f487d](https://github.com/BigWhaleLabs/obss-ipfs-uploader/pull/16/commits/d7f487d96020f9ae7729dcff73ebccc3dd16ebb8).